CustomText.Utilities.CognitiveSearch/Solution/CustomTextAppIndexer/CustomTextAppIndexer.Core/Services/CognitiveSearch/CustomTextIndexingService.cs

```diff
@@ -9,6 +9,13 @@ namespace Microsoft.Azure.CognitiveServices.Language.CustomText.Utilities.Search
 {
     public class CustomTextIndexingService : ICustomTextIndexingService
     {
+        public string customTextEndpoint, customTextKey, customTextModelID;
+        public CustomTextIndexingService(string _customTextEndpoint, string _customTextKey, string _customTextModelID)
```

Comment:
.net convention, arguments to functions or constructors should start with lowercase
_ underscore indicates a private class variable
i.e. remove the underscores from parameters
and add underscores to class attributes